### ISSUE TITLE
Equivalent of geoip_region_name_by_code

### ISSUE THREAD

**Issue by rubenbenjamin** [maxmind/MaxMind-DB-Reader-php#146]
Is there an equivalent of this function
```
<?php
$region = geoip_region_name_by_code('CA', 'QC');
if ($region) {
    echo 'Region name for CA/QC is: ' . $region;
}
?>
```

**Comment by oschwald** [comment]
No, there is not an equivalent of this function. The database now stores the subdivision name alongside the code and there isn't a reason to do a separate lookup. If you are doing the above outside of a GeoIP database lookup, I would suggest switching to something like [PHP ISO Codes](https://github.com/sokil/php-isocodes).
